README.md

```diff
@@ -74,6 +74,22 @@ print(qc_compiled.draw())
 
 **Detailed documentation and examples are available at [ReadTheDocs](https://mqt.readthedocs.io/projects/predictor).**
 
+## References
+
+In case you are using MQT Predictor in your work, we would be thankful if you referred to it by citing the following publication:
```

Comment:
```suggestion
In case you are using the MQT Predictor in your work, we would be thankful if you referred to it by citing the following publication:
```
Just wondering 🤔